experimental/index.html

```diff
@@ -100,21 +108,269 @@
+          >
+        </div>
+        <div class="block lg:hidden">
+          <button
+            class="flex items-center px-3 py-2 border rounded text-blue-600 border-teal-400 hover:text-slate-500 hover:border-white"
+          >
+            <svg
+              class="fill-current h-3 w-3"
+              viewBox="0 0 20 20"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <title>Menu</title>
+              <path d="M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z" />
+            </svg>
+          </button>
+        </div>
+
+        <div class="w-full block flex-grow lg:flex lg:items-center lg:w-auto">
+          <div class="text-sm lg:flex-grow">
+            <a
+              href="#responsive-header"
+              class="block mt-4 lg:inline-block lg:!mt-0 text-sky-400 hover:text-sky-600 hover:no-underline mr-4"
+            >
+              About
+            </a>
+          </div>
+          <div>
+            <div class="relative text-gray-600">
+              <input
+                type="search"
+                name="serch"
+                placeholder="Country"
+                class="bg-white h-10 px-5 pr-10 rounded-full text-sm focus:outline-none shadow-sm"
+              />
+              <button type="submit" class="absolute right-0 top-0 mt-3 mr-4">
+                <svg
+                  class="h-4 w-4 fill-current"
+                  xmlns="http://www.w3.org/2000/svg"
+                  xmlns:xlink="http://www.w3.org/1999/xlink"
+                  version="1.1"
+                  id="Capa_1"
+                  x="0px"
+                  y="0px"
+                  viewBox="0 0 56.966 56.966"
+                  style="enable-background: new 0 0 56.966 56.966"
+                  xml:space="preserve"
+                  width="512px"
+                  height="512px"
+                >
+                  <path
+                    d="M55.146,51.887L41.588,37.786c3.486-4.144,5.396-9.358,5.396-14.786c0-12.682-10.318-23-23-23s-23,10.318-23,23  s10.318,23,23,23c4.761,0,9.298-1.436,13.177-4.162l13.661,14.208c0.571,0.593,1.339,0.92,2.162,0.92  c0.779,0,1.518-0.297,2.079-0.837C56.255,54.982,56.293,53.08,55.146,51.887z M23.984,6c9.374,0,17,7.626,17,17s-7.626,17-17,17  s-17-7.626-17-17S14.61,6,23.984,6z"
+                  />
+                </svg>
+              </button>
+            </div>
+          </div>
+        </div>
       </div>
+    </nav>
+
+    <!-- Country Title -->
+    <div
+      class="flex flex-column relative dc-content mx-auto px-4 items-center py-12"
+    >
+      <h1 class="font-extralight text-7xl">United States</h1>
+    </div>
+
+    <!-- Development Goals -->
+    <div class="flex relative dc-content mx-auto text-slate-700">
+      <aside
+        class="h-screen sticky top-0 shrink-0 w-0 hidden md:block md:w-80 pt-4 pr-4"
+      >
+        <div
+          class="flex border-l-2 border-transparent hover:!border-slate-300 hover:text-blue-600 px-2 cursor-pointer mb-3"
```

Comment:
Since this is a long list, maybe use a javascript to create each one on the fly? https://jsfiddle.net/ncn7koqx/1/

Reply:
Good call- I extracted these to templates